lib/channel_manager.ts

```diff
@@ -12,6 +12,7 @@ import log from './util/log'
 import ChannelContract from './channel_contract'
 import PaymentManager from './payment_manager'
 import Web3 = require('web3')
+import { MachinomyOptions } from '../index'
```

Comment:
Circular dependency for the win!

Reply:
My bad. That's what I get for letting my IDE generate imports.